src/app/app.coffee

```diff
@@ -4,18 +4,27 @@
 # Doubtfire is modularised into many modules, as indicated by the directory
 # tree inside app/
 #
-#
+
+# Require the CSS file explicitly (or it could be defined as an entry-point too).
+require('../assets/styles/app.scss')
+
+# angular-markdown-filter expects window.showdown to exist
+window.showdown = require('showdown')
+# angular-ui-codemirror expects window.CodeMirror
+window.CodeMirror = require('codemirror')
```

Comment:
I kept it in the app.coffee because I don't like anything setting values on window hidden away in a module's dependencies somewhere. Actually what are you thoughts on moving vendor-dependencies.coffee to be next to app.coffee and require('./vendor-dependencies')'d?

I think it makes sense for vendor-dependencies to be defined as close to the entry point as possible to make the external easily noticeable